From 0868ba71c86f31866a48cac0f2495a400db12b34 Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Fri, 11 Jul 2014 10:47:26 -0500
Subject: [CHANGE 15/18] tests: vmstate static checker: remove a subsection
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <641a691f07b5b62b7cbbeddd6fc6291286b6c2a8.1405075344.git.amit.shah@redhat.com>
Patchwork-id: 59815
O-Subject: [RHEL6.6 qemu-kvm PATCH 15/18] tests: vmstate static checker: remove a subsection
Bugzilla: 1118718
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Signed-off-by: Amit Shah <amit.shah@redhat.com>
Reviewed-by: Juan Quintela <quintela@redhat.com>
Signed-off-by: Juan Quintela <quintela@redhat.com>
(cherry picked from commit aa2a12bb824170a0dae94f0d39bbd3b61add038f)
Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 tests/vmstate-static-checker-data/dump2.json | 13 -------------
 1 file changed, 13 deletions(-)

Signed-off-by: jen <jen@redhat.com>
---
 tests/vmstate-static-checker-data/dump2.json | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/tests/vmstate-static-checker-data/dump2.json b/tests/vmstate-static-checker-data/dump2.json
index 05b8fce..6f8a617 100644
--- a/tests/vmstate-static-checker-data/dump2.json
+++ b/tests/vmstate-static-checker-data/dump2.json
@@ -836,19 +836,6 @@
                           "size": 1
                         }
                       ]
-                    },
-                    {
-                      "name": "fdrive/media_rate",
-                      "version_id": 1,
-                      "minimum_version_id": 1,
-                      "Fields": [
-                        {
-                          "field": "media_rate",
-                          "version_id": 0,
-                          "field_exists": false,
-                          "size": 1
-                        }
-                      ]
                     }
                   ]
                 }
-- 
1.9.3

